book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :name, :author, :rentals
+
```

Comment:
- [ ] Also here, to make the association complete, we need to make sure that the instance variables have setters and getters. I notice you have not added the `getter` or `setter` for the `:title`, Please add the `attr_accessor` for the `:title` instance variable.